go/client/cmd_chat_listchannels.go

```diff
@@ -70,9 +70,12 @@ func (c *CmdChatListChannels) Run() error {
 
 	ui.Printf("Listing channels on %s:\n\n", c.tlfName)
 	for _, c := range listRes.Convs {
-		convLine := fmt.Sprintf("#%s", c.Channel)
+		// c.Channel and c.Headline are user generated content and could
+		// technically contain formatting directives so we don't use
+		// fmt.Sprintf to concatenate them.
+		convLine := "#" + c.Channel
 		if c.Headline != "" {
-			convLine += fmt.Sprintf(" [%s]", c.Headline)
+			convLine += " [" + c.Headline + "]"
 		}
 		if c.CreatorInfo != nil {
 			convLine += fmt.Sprintf(" (created by: %s on: %s)", c.CreatorInfo.Username,
```

Comment:
is the real bug on line 81 `ui.Printf(convLine + "\n")` ? do `ui.Printf("%s\n", convLine)` instead

Reply:
ah good catch!

Reply:
```
$ kb chat list-channels --topic-type dev team1
Running `go install github.com/keybase/client/go/keybase`... Done.
▶ WARNING Running in devel mode
Listing channels on team1:

#^% (created by: test1 on: 2018-12-21)
```